src/consts/misc.ts

```diff
@@ -0,0 +1,8 @@
+export const SQUID_ADDRESS = {
+	mainnet: '0xce16F69375520ab01377ce7B88f5BA8C48F8D666',
+	testnet: '0xC3468a191Fe51815b26535ED1F82C1f79e6Ec37D',
+}
+
+export const PEANUT_SALT = '0x70adbbeba9d4f0c82e28dd574f15466f75df0543b65f24460fc445813b5d94e0'
+export const ANYONE_WITHDRAWAL_MODE = '0x0000000000000000000000000000000000000000000000000000000000000000'
+export const RECIPIENT_WITHDRAWAL_MODE = '0x2bb5bef2b248d3edba501ad918c3ab524cce2aea54d4c914414e1c4401dc4ff4'
```

Comment:
Good

Reply:
Could read this from onchain contract, but I don't think we're gonna change this often so this is great

Reply:
Yeah, agreed. And for example for the v4.2 contract these never change